samples/qt/README.md

```diff
@@ -0,0 +1,114 @@
+<!-- (C) Copyright 2025, SECO Mind Srl
+
+SPDX-License-Identifier: Apache-2.0 -->
+
+# Astarte Device SDK C++ - Qt Example
+
+This example demonstrates how to use the **Astarte Device SDK for C++** in a **Qt** application.
+
+
+## 📁 Example Location
+
+```
+astarte-device-sdk-cpp/samples/qt
+```
+
+
+## 🛠️ Requirements
+
+- **Qt**
+
+  ⚠️ *Minimum required version: 6.8.1*
+
+- **CMake**
+
+    ⚠️ *Minimum required version: 3.19*
+
+- **C++ Compiler**
+
+    ⚠️ *Requires support for at least C++17*
+
+- **gRPC and dependencies** (including protobuf)
+
+  ⚠️ *Minimum required gRPC version: 1.69.0*
+
+  ⚠️ *Minimum required protobuf version: 3.15*
+
+- **Rust**
+
+  ⚠️ *Minimum required version: 1.78.0*
```

Comment:
I would not place a minimum required version of Rust here. It's not a dependency of this project but of the Astarte Message hub.
You can set a requirement for the minimum version of the Astarte Message hub if you wish to do so (use the latest release in that case).